src/WebJobs.Script.WebHost/App_Start/WebHostResolver.cs

```diff
@@ -239,6 +251,16 @@ private TraceWriter CreateDefaultTraceWriter(WebHostSettings settings)
             return new CompositeTraceWriter(new[] { systemTraceWriter, fileTraceWriter });
         }
 
+        private ILoggerFactory CreateDefaultLoggerFactory(WebHostSettings settings)
+        {
+            var config = GetScriptHostConfiguration(settings);
+            var clientFactory = new ScriptTelemetryClientFactory(_settingsManager.ApplicationInsightsInstrumentationKey, config.ApplicationInsightsSamplingSettings, config.LogFilter.Filter);
+            var loggerFactory = new LoggerFactory();
+            loggerFactory.AddApplicationInsights(clientFactory);
```

Comment:
We should only add this LoggerProvider if `ApplicationInsightsInstrumentationKey` wasn't null.